content/docs/command-reference/push.md

```diff
@@ -157,37 +157,51 @@ a [pipeline](/doc/command-reference/pipeline) has been setup with these
 [stages](/doc/command-reference/run):
 
 ```dvc
-$ dvc pipeline show
-data/Posts.xml.zip.dvc
-Posts.xml.dvc
-Posts.tsv.dvc
-Posts-test.tsv.dvc
-matrix-train.p.dvc
-model.p.dvc
-Dvcfile
+$ dvc dag
+             +-------------+
+             | prepare.dvc |
+             +-------------+
+                    *
+                    *
```

Comment:
@jorgeorpinel The [dag example](https://dvc.org/doc/command-reference/dag#examples) in cmd ref doesn't have `.dvc` in stage name it prints. Should I modify `dvc dag` ?